src/converter/modules/blocks/create-shade-box.js

```diff
@@ -7,11 +7,26 @@ import { replaceElement, toBlock } from '../utils/dom-utils.js';
 export default function createShadeBox(document) {
   Array.from(document.querySelectorAll('.sp-wrapper')).forEach(
     (shadeBoxElement) => {
-      const children = Array.from(shadeBoxElement.childNodes);
-      replaceElement(
-        shadeBoxElement,
-        toBlock('shade-box', [[children]], document),
+      const section = shadeBoxElement.parentElement;
+      const shadeBoxSection = document.createElement('div');
+      const subsequenSection = document.createElement('div');
+      section.after(shadeBoxSection);
+
+      let nextPointer = shadeBoxElement.nextElementSibling;
+      if (nextPointer) {
+        shadeBoxSection.after(subsequenSection);
+        while (nextPointer) {
+          const next = nextPointer.nextElementSibling;
+          subsequenSection.append(nextPointer);
+          nextPointer = next;
+        }
+      }
```

Comment:
Might we need to do this in multiple places; extracting a block into a dedicated section?

If so, might be good to move this into a helper method for re-use?